request_data.py

```diff
@@ -0,0 +1,54 @@
+import emission.core.get_database as edb
+import bson.json_util as bju
+import requests
+import json
+import argparse
+import logging
+
+parser = argparse.ArgumentParser()
+parser.add_argument("from_date",
+        help="from_date in the format of YYYY-MM-DD")
```

Comment:
hmm, I thought this would be local time? When I tested this locally, I entered 2016-06-30 as `from_date` and the first returned timeseries entry for the phones had `write_local_dt': {u'hour': 0, u'month': 6, u'second': 24, u'weekday': 3, u'year': 2016, u'timezone': u'America/Los_Angeles', u'day': 30, u'minute': 4}` which seems to me the `from_date` was interpreted as local time